### DISCUSSION TITLE
How to add TMDD in the model of antibody?

### DISCUSSION THREAD

**Discussion by woniujy** [Open-Systems-Pharmacology/Forum#1513] [Questions & Problems]
Hi,
I am a newbie here. I am working on PBPK model of an antibody. I have only find parameters related to FcRn to be adjusted. Could you please tell me how to add TMDD in the model? Thanks very much.

**Comment by Christoph27** [comment]
Hi,
For a typical TMDD model you need to extend the PBPK model in MoBi.
Typically, you can do the following steps to extend the model with the processes involved in TMDD:
1)	Set up the base PBPK model in PK-Sim including the target protein as “Protein Binding Partner” (define the expression profile at Individual/Expression profile and the binding properties [dissociation & koff rate constant] at the drug molecule)
2)	Export this model to MoBi and extend the model in Mobi with: 
    a) A target turnover reaction (incl. the turnover rate constant)
    b) A reaction representing internalization/degradation of the drug-target complex (with the respective rate constant)

There is a tutorial where these steps (among others) are described in more detail: 
https://github.com/Open-Systems-Pharmacology/ISOP-webinar---PK-Sim-MoBi-introduction

Unfortunately, the tutorial is a bit outdated, e.g., the synthesis part of the target turnover can now be implemented simpler since nowadays there is a parameter “Start amount” at the target molecule. The total turnover could nowadays look something like this:

<img width="283" alt="image" src="https://user-images.githubusercontent.com/26439150/235674971-69cfffd9-a143-4253-ad2a-838bfc873cb1.png">


Also, molecules to localize reactions (as e.g. “is interstitial”) are nowadays not needed anymore since you can define container criteria directly at the reactions you build.  

Hope that helps …

**Reply by StephanSchaller** [reply]
For an up-to-date tutorial you can always go to: https://learn-esqlabs.talentlms.com/ (free for academia)
--> course: "[Antibody Therapy in Oncology](https://esqlabs.shop/products/antibody-therapy-in-oncology-2)" (for industry)
